src/Features/Core/Portable/LanguageServices/SymbolDisplayService/SymbolDisplayClassificationHelper.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.DocumentationComments;
```

Comment:
You came by my office to tell me I was trolling you, but I wasn't. Note that the usings below this comment aren't sorted. 😛